### PR TITLE
Update BDLaethf29.xml

### PR DESCRIPTION
**Pull request by SusanneHummel**
- added div edition
- updated titles
- changes for standardization
- minor corrections

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethf29.xml [hunk 1]
```diff
@@ -359,8 +351,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <material key="parchment"/>
                                     <p>A hole on <locus target="#54"/>, presumably from parchment
                                         making </p>
-                                    <p>A tear carefully amended on <q xml:lang="gez">ዘእኁድ</q>
-                                            (<locus target="#74r"/>)<locus target="#42"/>
+                                    <p>A tear carefully amended on <locus target="#42"/>
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethf29.xml:354]: I see that according to the guidelines everything concerning the material and its characteristics can go under `<material key="parchment"/>`(https://betamasaheft.eu/Guidelines/?q=condition&id=objectDescription).
However, I have so far discriminated between holes resulting from parchment making (which I encoded here) and tears/holes repaired (@Gnisci once corrected my "amended" into "repaired") by someone more or less carefully, which I encoded under Condition (in my view this seems to fit better that kind of feature).
We can discuss this here or in the next meeting and find a consistent solution.

File: OxfordBodleian/Juel-Jensen/BDLaethf29.xml [hunk 2]
```diff
@@ -878,10 +869,27 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
+
+            <div type="edition">
+                <div type="textpart" corresp="#ms_i6" xml:lang="gez" xml:id="PrayerText">
+                    <ab><pb n="118r"/>
+                        <hi rend="rubric"> በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ በ</hi>ስመ፡ ፫አስማት፡
+                            አአትብ፡<pb n="118v"/> ገጽየ፡ አአትብ፡ ድኅሬየ፡ በዘቦቱ፡ ተወኪልየ፡ ዕመውዕ፡ ፀርየ፡ በስመ፡ ፫አስማት፡
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethf29.xml:883]: space between numeral and noun, also in the following lines.

**Review by MassimoVilla — CHANGES_REQUESTED** [review]

File: OxfordBodleian/Juel-Jensen/BDLaethf29.xml [hunk 3]
```diff
@@ -7,8 +7,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Sayfa śǝllāse, Malkǝʾa śǝllāse, Naʿā ḫabeya ʾo-ʾǝgziʾǝya ʾIyasus,
-                    Hymns</title>
+                <title>, Malkǝʾa śǝllāse, Naʿā ḫabeya ʾo-ʾǝgziʾǝya ʾIyasus, Hymns</title>
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethf29.xml:10]: Do you want to keep the first title line?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: OxfordBodleian/Juel-Jensen/BDLaethf29.xml (hunk 1 above)
Comment (SusanneHummel) [reply OxfordBodleian/Juel-Jensen/BDLaethf29.xml:354]: Holes are commonly already repaired during the parchment making process. See here: Winslow 2015, 109. If there are no hints for any later repair (like synthetic therads), I would group them here.

**Pull request merged**